### DISCUSSION TITLE
How to mimic Chakra UI's `colorScheme` prop?

### DISCUSSION THREAD

**Discussion by asterikx** [chakra-ui/panda#882] [Q&A] [answered]
Is there a recommended way to re-create Chakra UI's [`colorScheme` prop ](https://chakra-ui.com/docs/styled-system/customize-theme#customizing-theme-tokens)with Panda CSS?

If there is only a limited number of colors, as in the case of the [Alert](https://chakra-ui.com/docs/components/alert/usage) component, I can introduce a `data-status` attribute and style my components based on it (using green for `data-status=success` etc.). Is this the recommended solution?

How about arbitrary color schemes?

**Comment by anubra266** [comment, marked as answer]
@asterikx https://panda-css.com/docs/guides/virtual-color

**Reply by asterikx** [reply]
@anubra266 Thank you! I must have missed that when I was stepping through the docs.

**Reply by asterikx** [reply]
@anubra266 can a virtual color also be applied to recipes?

In case of the Alert component, the color from the `colorScheme` needs to be [applied to multiple parts](https://github.com/chakra-ui/chakra-ui/blob/00a06704e5ba609f5b8f4787873dd7c72d56636e/packages/components/theme/src/components/alert.ts#L15-L43).
